.github/workflows/ci.yml

```diff
@@ -412,7 +417,7 @@ jobs:
     steps:
       - name: Create Dev Drive using ReFS
         run: |
-          $Volume = New-VHD -Path C:/uv_dev_drive.vhdx -SizeBytes 10GB |
+          $Volume = New-VHD -Path C:/uv_dev_drive.vhdx -SizeBytes 12GB |
```

Comment:
I tweaked this to match the size used in the `build binary | windows` job to try to help resolve this failure: https://github.com/actions/runner/issues/2491

Now we're hitting this other error which I'm not sure how to resolve: https://github.com/astral-sh/uv/actions/runs/9768734370/job/26966858668?pr=4734

```
  = note:    Creating library E:\uv\target\debug\deps\uv.lib and object E:\uv\target\debug\deps\uv.exp

          LINK : fatal error LNK1318: Unexpected PDB error; LIMIT (12) 'E:\uv\target\debug\deps\uv.pdb'
```

Reply:
That insane error is also an indication that the disk is full afaik, you can make it even bigger.